frontend/src/model/releases/ReleaseDisplay.tsx

```diff
@@ -22,10 +22,12 @@ export default function ReleaseDisplay({
   model,
   release,
   latestRelease,
+  displayReviewBanner = true,
 }: {
```

Comment:
Not a change you made in this PR, but can we pull this prop type out into a `ReleaseDisplayProps` type to align with the rest of the app?